libs/vm-modules/include/vm_modules/utilities.hpp

```diff
@@ -0,0 +1,52 @@
+#pragma once
```

Comment:
I think we can do better than `utilities` as a name for this file. How about `estimator_use`? (or `use_estimator`, if you end up renaming it)